options/comments.md

```diff
@@ -93,8 +93,27 @@ Prevents warnings due to unspecified tags from being reported in comments within
 }
 ```
 
-Override TypeDoc's supported block tags, emit warnings for any tags not listed here.
-This option will be set by `tsdoc.json` if present.
+This specifics all of the [JSDoc block tags](../guides/tags.md#block-tags) that TypeDoc considers to be valid.
```

Comment:
This link is broken. Eleventy doesn't rewrite links to markdown files to be the actual output file

Reply:
can you clarify, should I just remove the .md? or use a non relative url?

Reply:
An absolute URL is the way I'd recommend, similar to https://github.com/TypeStrong/typedoc-site/blob/master/tags/eventProperty.md?plain=1#L30